cognite/neat/graph/stores/_base.py

```diff
@@ -185,44 +140,31 @@ def reinit_graph(self):
             self.internal_storage_dir,
         )
 
-    def close(self):
+    def close(self) -> None:
         """Closes the graph."""
-        if self.rdf_store_type == RdfStoreType.OXIGRAPH:
-            try:
-                if self.graph:
-                    self.graph.store._inner.flush()  # type: ignore[attr-defined]
-                    self.graph.close(True)
-            except Exception as e:
-                logging.debug("Error closing graph: %s", e)
-
-    def restart(self):
+        # Can be overridden in subclasses
+        return None
+
+    def restart(self) -> None:
         """Restarts the graph"""
-        if self.rdf_store_type == RdfStoreType.OXIGRAPH:
-            self.close()
-            self.reinit_graph()
-            logging.info("GraphStore restarted")
+        # Can be overridden in subclasses
+        return None
 
-    def import_from_file(self, graph_file: Path, mime_type: str = "application/rdf+xml", add_base_iri: bool = True):
+    def import_from_file(
```

Comment:
Using enums instead of pure string:

```suggestion
    def import_from_file(
        self, graph_file: Path, mime_type: StrEnum = MIMETypes.xml, add_base_iri: bool = True
    ) -> None
```